blocks/columns/columns.css

```diff
@@ -267,4 +267,140 @@
   }
 }
 
+.columns.columns-image-right {
```

Comment:
@skumawat1 many rules are common in both variations, like size, color, radius but rules are duplicated.
Suggest not to duplicate rules.